ext_localconf.php

```diff
@@ -1,9 +1,20 @@
 <?php
+
+use TYPO3\CMS\Extbase\Utility\ExtensionUtility;
```

Comment:
Do not use `use` in ext_localconf, see https://docs.typo3.org/m/typo3/reference-coreapi/10.4/en-us/ExtensionArchitecture/ConfigurationFiles/Index.html#configuration-files-ext-tables-php-ext-localconf-php (Box "Important")